index.html

```diff
@@ -5,17 +5,6 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta property="og:title" content="GMX | Decentralized Perpetual Exchange" />
-    <meta property="og:site_name" content="GMX" />
-    <meta property="og:image" content="https://gmx.io/og.png" />
-    <meta
-      name="og:description"
-      content="Trade spot or perpetual BTC, ETH, AVAX and other top cryptocurrencies with up to 100x leverage directly from your wallet on Arbitrum and Avalanche."
-    />
-    <meta
-      name="description"
-      content="Trade spot or perpetual BTC, ETH, AVAX and other top cryptocurrencies with up to 100x leverage directly from your wallet on Arbitrum and Avalanche."
-    />
```

Comment:
why removing this?

Reply:
Hmm, on one hand, this is a duplication of the SEO component: https://github.com/gmx-io/gmx-interface/blob/vite-build/src/components/Common/SEO.tsx.

On the other hand, it may be useful to have some meta tags even in the static HTML.

Another thought is that all modern crawlers should execute JavaScript.

What do you think?

Reply:
Discussed and decided to bring them back to keep seo stable